charms/trusty/clearwater-bono/lib/chef_solo_install

```diff
@@ -1,22 +1,19 @@
 #!/bin/bash
 
 # This covers installing chef-solo and the Clearwater chef recipes
-# Install needed libraries
-apt-get -y install git libxml2-dev libxslt1-dev wget curl
-
-# Install chef
-$CHARM_DIR/lib/chef-install.sh -v 11.6.0
+# Install chef and needed tools/libraries
+apt-get -y install chef git libxml2-dev libxslt1-dev wget curl
 
 # Pull down the Clearwater chef recipes
 if [ ! -d /home/ubuntu/chef-solo ]
 then
   mkdir /home/ubuntu/chef-solo
-  git clone --recursive -b master git://github.com/Metaswitch/chef.git /home/ubuntu/chef-solo
+  git clone --recursive git://github.com/Metaswitch/chef.git /home/ubuntu/chef-solo
 fi
 
 # Update the chef recipes
 cd /home/ubuntu/chef-solo
-git checkout release-70
+git checkout stable
```

Comment:
We should check out release-90, to keep this stable in future without having to constantly update the charms.